agent/tool-scripts/postprocess/openshift-metrics-postprocess

```diff
@@ -7,8 +7,8 @@ output_file = sys.argv[2]
 in_csv = []
 metrics=[]
 
-with open(input_file, 'r') as f:
-    reader = csv.reader(f)
+with open(input_file, 'r') as file:
+    reader = csv.reader(null_bytes.replace('\0', '') for null_bytes in file)
```

Comment:
Can you add a comment describing why you have to process null bytes?  What is emitting null bytes to begin with?